docs/platforms/react-native/migration/v5-to-v6.mdx

```diff
@@ -0,0 +1,253 @@
+---
+title: Migrate from 5.x to 6.x
+sidebar_order: 100
+description: "Learn how to migrate from version 5 to version 6 of the Sentry React Native SDK"
+---
+
+The main goal of version 6 of the Sentry React Native SDK, is to bring compatibility with Sentry JavaScript version 8 and to improve our performance monitoring APIs and integrations. This version is breaking because of the JavaScript SDK dependency upgrade to version 8 and because we've removed deprecated APIs and restructured npm package contents.
```

Comment:
l: NIT
```suggestion
The main goal of version 6 of the Sentry React Native SDK, is to bring compatibility with Sentry JavaScript version 8 and to improve our performance monitoring APIs and integrations. This version is introducing breaking changes because of the JavaScript SDK dependency upgrade to version 8 and because we've removed deprecated APIs and restructured npm package contents.
```